### PR TITLE
docs(contributing guide): fixed broken links in contributing outline

### PR DESCRIPTION
**Pull request by gefgu**
**What**: These 3 links are not working in the [contribution guidelines](https://github.com/patternfly/patternfly-react/blob/main/CONTRIBUTING.md).
screenshot

Closes: #8041

### CODE REVIEW

**Comment by tlabaj** [comment]
@gefgu can you please link an issue to this PR for tracking purposes.  thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,9 +22,9 @@
   - [Code Consistency](#code-consistency)
   - [Code Contribution Guidelines](#code-contribution-guidelines)
 - [Becoming a Maintainer](#becoming-a-maintainer)
-  - [How do I become a maintainer?](how-do-i-become-a-maintainer?)
-  - [How do I lose maintainers status?](how-do-i-lose-maintainers-status?)
-  - [Quick Tips for New Maintainers](quick-tips-for-new-maintainers)
+  - [How do I become a maintainer?](#how-do-i-become-a-maintainer?)
+  - [How do I lose maintainers status?](#how-do-i-lose-maintainers-status?)
```
Comment (thatblindgeye) [inline CONTRIBUTING.md:26]: We would also need to remove the trailing question marks `?` at the end of the URLs here.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Thanks for catching this! In addition to the above comment and my own change request below, a more fitting commit type in your commit message could be `docs` instead of `fix`. The comment at https://github.com/patternfly/patternfly/pull/5102#pullrequestreview-1112868383 actually goes into some difference between them.

**Comment by gefgu** [comment]
@thatblindgeye thanks for guiding me on how to improve my commit messages. Already implemented the changes :)

**Review by thatblindgeye — APPROVED** [review]

**Review by tlabaj — APPROVED** [review]

**Pull request merged**